ontology/yaml/resources/ELECTRICAL/entity_types/ABSTRACT.yaml

```diff
@@ -183,3 +183,12 @@ SRC2:
   - source2_phase3_phase1_over_voltage_status
   - source2_over_voltage_status
   - source2_lost_power_alarm
+
+BBM:
```

Comment:
Please remove this type (see comment on UPS.yaml)

Reply:
note ma'am, will do. thanks